src/sentry/static/sentry/app/utils/projects.tsx

```diff
@@ -283,12 +370,24 @@ class Projects extends React.Component {
 
 export default withProjects(withApi(Projects));
 
+type FetchProjectsOptions = {
+  slugs?: string[];
+  cursor?: State['nextCursor'];
+  search?: State['prevSearch'];
+  prevSearch?: State['prevSearch'];
```

Comment:
@billyvg Seems like you could do `Partial<Pick<State, 'nextCursor' | 'prevSearch' | 'prevSearch'>>` here